internal/cli/cmd/cluster/grpcproxy.go

```diff
@@ -109,9 +114,13 @@ func (g *grpcProxy) handler(srv interface{}, serverStream grpc.ServerStream) err
 		return status.Errorf(codes.Internal, "failed to connect to backend: %v", err)
 	}
 
+	clientStreamDescForProxying := &grpc.StreamDesc{
```

Comment:
Where is this used?

Reply:
Here:
https://github.com/namespacelabs/foundation/pull/1207/files/96e07093e2d8485f7c8efbb128cd0e137499e215#diff-394e8fb96279ff1d4b57f11cc8c927841543f6a7f08c1fc2a32499ec0732c120R124